contrib/experimental/great_expectations_experimental/expectations/expect_column_kurtosis_to_be_between.py

```diff
@@ -52,102 +82,167 @@ class ColumnKurtosis(ColumnMetricProvider):
-    #
-    #     dependencies = super()._get_evaluation_dependencies(
-    #         metric=metric,
-    #         configuration=configuration,
-    #         execution_engine=execution_engine,
-    #         runtime_configuration=runtime_configuration,
-    #     )
-    #
-    #     table_domain_kwargs = {
-    #         k: v for k, v in metric.metric_domain_kwargs.items() if k != "column"
-    #     }
-    #
-    #     dependencies.update(
-    #         {
-    #             "table.row_count": MetricConfiguration(
-    #                 "table.row_count", table_domain_kwargs
-    #             )
-    #         }
-    #     )
-    #
-    #     if isinstance(execution_engine, SqlAlchemyExecutionEngine):
-    #         dependencies["column_values.nonnull.count"] = MetricConfiguration(
-    #             "column_values.nonnull.count", metric.metric_domain_kwargs
-    #         )
-    #
-    #     return dependencies
+    @metric_value(engine=SqlAlchemyExecutionEngine)
+    def _sqlalchemy(
+        cls,
+        execution_engine: "SqlAlchemyExecutionEngine",
+        metric_domain_kwargs: Dict,
+        metric_value_kwargs: Dict,
+        metrics: Dict[Tuple, Any],
+        runtime_configuration: Dict,
+    ):
+        (
+            selectable,
+            compute_domain_kwargs,
+            accessor_domain_kwargs,
+        ) = execution_engine.get_compute_domain(
+            metric_domain_kwargs, MetricDomainTypes.COLUMN
+        )
+
+        column_name = accessor_domain_kwargs["column"]
+        column = sa.column(column_name)
+        sqlalchemy_engine = execution_engine.engine
+        dialect = sqlalchemy_engine.dialect
+
+        column_mean = _get_query_result(
+            func=sa.func.avg(column * 1.0),
+            selectable=selectable,
+            sqlalchemy_engine=sqlalchemy_engine,
+        )
+
+        column_count = _get_query_result(
+            func=sa.func.count(column),
+            selectable=selectable,
+            sqlalchemy_engine=sqlalchemy_engine,
+        )
+
+        # sql dialects are inconsistent about stddev vs stddev_samp
+        # try to use stddev_samp and correct by multipying by (N-1)/N
+        if dialect.name.lower() == "mssql":
+            standard_deviation = sa.func.stdev(column)
+        else:
+            standard_deviation = sa.func.stddev_samp(column)
+
+        column_std = _get_query_result(
+            func=standard_deviation,
+            selectable=selectable,
+            sqlalchemy_engine=sqlalchemy_engine,
+        )
+
+        column_fourth_moment = _get_query_result(
+            func=sa.func.sum(sa.func.pow(column - column_mean, 4)),
+            selectable=selectable,
+            sqlalchemy_engine=sqlalchemy_engine,
+        )
+
+        column_kurtosis = (
+            column_fourth_moment
+            / (column_std ** 4)
+            / column_count
+            * (column_count) ** 2
+            / ((column_count - 1) ** 2)
+        )
+        return column_kurtosis - 3
```

Comment:
Why only implement Fisher and not Pearson as well?

Reply:
Was just following the previous work here, which also did weird-but-not-catastrophic things like assign within local scope to 'abs'. 

The difference between these things is literally 'subtract three', so sure can add.